src/entity/StudentExam.ts

```diff
@@ -47,10 +37,10 @@ export class StudentExam {
     })
     letterGrade: LetterGrade;
 
-    @ManyToOne(type => Student, student => student.studentExams)
+    @ManyToOne(type => Student, student => student.studentExams, {eager:true})
```

Comment:
I like what `eager: true` does in the HTTP request -- it isn't set for all the relations though (i.e., when I GET notes, I don't see what students they're related to). Maybe another ticket to do this for all entity relations?